src/main/java/org/cirdles/topsoil/app/utils/SVGSaver.java

```diff
@@ -45,15 +74,36 @@ public void save(Document svgDocument) {
         fileChooser.getExtensionFilters().add(new FileChooser.ExtensionFilter("SVG Image", "*.svg"));
         File svgFile = fileChooser.showSaveDialog(null);
 
+        return svgFile;
```

Comment:
If this can return `null`, I'd rather you look into using [`Optional`](https://docs.oracle.com/javase/8/docs/api/java/util/Optional.html). Using `Optional` would also eliminate the need for `noFileSpecified`.